jenkins-pipelines/features-scale-cluster-test.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    params: params,
+
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'grow_cluster_test.GrowClusterTest.test_grow_x_to_y',
```

Comment:
After the basic test passes, we need to extend this test to support nemesis.
If nemesis set in the yaml it should start nemesis (same as in longevity)